client/web/src/views/ChartViewContent/charts/line/helpers/get-range-with-padding.ts

```diff
@@ -12,5 +12,6 @@ export function getRangeWithPadding(range: [number, number], paddingCoefficient:
     const [min, max] = range
     const increment = ((max - min) * paddingCoefficient) / 2
 
-    return [min - increment, max + increment]
+    // Minimal value for insight line chart can't be less than 0
+    return [Math.max(min - increment, 0), max + increment]
```

Comment:
Is there any reason we should not just use `min` without the padding so that hypothetical charts that have negative values still work? E.g. some kind of change rate chart
```suggestion
    return [min, max + increment]
```

Reply:
Don't add padding (`- increment`) if min is positive and within `increment` distance from 0? Would make sense to me visually as well to clamp to 0 within that range, otherwise to have balanced top and bottom padding

Reply:
@felixfbecker origin idea was about adding the padding at the top and bottom of the line chart. So if we want to use just `min` it would be mean that we don't want to have a bottom padding for all charts. 

but this also might work 
>Don't add padding (- increment) if min is positive and within increment distance from 0?

Reply:
Yeah, my suggestion was to not have padding at the bottom. I feel like that could work more nicely than having padding, because it would mean the lowest point would sit right on the X-axis, which gives it more of a feeling that the line is "cut off" there (which it is, semantically) as opposed to (in mathematical terms) the function of the chart only being defined for a specific interval.

But let's not get too held up by this, it's a very minor detail.